telemetrylib/TCP.cpp

```diff
@@ -62,14 +66,38 @@ public slots:
         {
             QTcpSocket* sender = static_cast<QTcpSocket*>(QObject::sender());
             _sockets.removeOne(sender);
-            connected = false;
+            connection = false;
             emit connectionStatusChanged();
         }
     }
 private:
-    QTcpServer _server;     //the TCP socket
-    QList<QTcpSocket*> _sockets;    //The list of TCP socket
-    bool connected = false;     //This is used when getConnectionStatus() is called
+    QTcpServer _server;
+    QList<QTcpSocket*> _sockets;
+    std::atomic<bool> connection = false;
+    std::thread *t;
+    std::atomic<bool> isConnected=false;
+
+    /**
+     * creates a thread that pings the Chase car's static IP to check connection
+     */
+    void checkConnection() {
+        QTcpSocket sock;
+        while(!isConnected) {
+            sock.connectToHost("196.168.1.16", 22);
+            bool connected = sock.waitForConnected(500);
```

Comment:
```suggestion
            bool connected = sock.waitForConnected(4000);
```
Let's wait for a hair less than the amount of time as the engineering data distributor timeout (see https://github.com/badgerloop-software/chase-car-dashboard/pull/30/files#diff-a63ba590aa3df476cd2406d359de28cfc547b3db0994d58238232061522d77c7R56)